draft-packages/form/KaizenDraft/Form/RadioGroup/RadioGroup.spec.tsx

```diff
@@ -65,4 +65,25 @@ describe("<RadioGroup /> ", () => {
       expect(queryByText(title)).toBeTruthy()
     })
   })
+  describe("accessibilty", () => {
+    it("should markup aria label when supplied labelId", () => {
+      const labelId = "labelId"
+      const { getByText, container } = render(
+        <RadioGroup
+          labelText="Label title"
+          labelId={labelId}
+          automationId="test"
+        />
+      )
+      expect(container.firstElementChild?.getAttribute("role")).toEqual(
```

Comment:
Nit: Just had a thought - could we perhaps use `getByRole("radiogroup")` (https://testing-library.com/docs/queries/byrole/)

Reply:
I knew I was forgetting something! Sure I'll give it a go

Reply:
Done.